src/Test/QuickCheck/StateMachineModel/Example.hs

```diff
@@ -67,12 +67,12 @@ transitions m cmd _ = case cmd of
   Write ref i -> update m (unRef ref) i
   Inc   ref   -> update m (unRef ref) ((m ! ref) + 1)
 
-postconditions :: Model -> MemStep resp  -> resp -> Bool
+postconditions :: Model -> MemStep resp  -> resp -> Property
 postconditions m cmd resp = case cmd of
-  New         -> True
-  Read ref    -> m  ! ref == resp && m' == m
-  Write ref i -> m' ! ref == i
-  Inc   ref   -> m' ! ref == m ! ref + 1
+  New         -> property $ True
+  Read ref    -> m  ! ref === resp .&&. m' == m
+  Write ref i -> property $ m' ! ref == i
```

Comment:
Yeah, I'm still playing around with the output. The `===` combinator adds a `counterexample` if the equality test fails, but they seem end up flooding the screen with newlines for some reason I don't get yet...